src/components/Form.js

```diff
@@ -7,23 +7,25 @@ import Collapsable from './Collapsable';
 
 class Form extends React.Component {
     render() {
-        const {actionToPerform} = this.props;
+        const { actionToPerform, actionColor, palette } = this.props;
         return (
             <section className="main__form">
                 <form action="" method="post">
-                    <Collapsable 
+                    <Collapsable
                         fieldset={'design'}
                         title={'Diseña'}
                         icon={'far fa-object-ungroup'}
-                        component={<Design />}
-                        >
+                        component={<Design actionColor={actionColor} palette={palette}/>}
+                    >
                     </Collapsable>
 
                     <Collapsable
                         fieldset={'fill'}
                         title={'Rellena'}
                         icon={'far fa-keyboard'}
+
                         component={<Fill actionToPerform={actionToPerform} name={this.props.name} job={this.props.job} email={this.props.email} tel={this.props.tel} linkedin={this.props.linkedin} github={this.props.github} />}
```

Comment:
It would be better to put these in the destructuring in line 10, instead of writing 'this.props' each time.